### ISSUE TITLE
Guild Wars 2 not working

### ISSUE THREAD

**Issue by jotkah** [doitsujin/dxvk#4553]
Hi Folks,

Guild Wars 2 does not start with dx-vk 2.5.2 (wine-staging 10.0 rc3). I got a "stuck" screen and dozens of messages on my console:

info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Color space:  VK_COLOR_SPACE_SRGB_NONLINEAR_KHR
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR (dynamic: no)
info:    Buffer size:  1920x1080
info:    Image count:  3

When i switch to "buildin" DLLs, everything works fine (without 3d acceleration). Other games also work fine on the same laptop. So it seems to be an issue between gw2 and dx-vk?

I have a nVidia 1060 graphics card. Driver Version 535.183.01 (Linux Mint).

What can i do now?

Greetings,
Joli

**Comment by Blisto91** [comment]
Could you attach a full dxvk log or console output?
I gave GW2 a quick test on Intel a few days ago without issue.

**Comment by jotkah** [comment]
Thank you for your quick reply. I hope you can follow me 'cause of my bad english.

Please have a look at the attachments.

There are two wine logs: One log with the dx-vk native DLL settings (not working, i have pressed CTRL-C after a while), one with the buildin DLL settings (working, but no 3D support).

My system is an "out of the box" Linux Mint Debian Based. wine 10.0-rc3 has been taken from the official Debian repository. apt says that everything is up to date.

I can create log files from another game on same platform for comparison, just let me know if you need this.

This is no urgent issue. Maybe you want to wait until Santa is back to north pole ;-)

[CefHost_d3d11.log](https://github.com/user-attachments/files/18246833/CefHost_d3d11.log)
[CefHost_dxgi.log](https://github.com/user-attachments/files/18246834/CefHost_dxgi.log)
[Gw2-64_d3d9.log](https://github.com/user-attachments/files/18246835/Gw2-64_d3d9.log)
[Gw2-64_d3d11.log](https://github.com/user-attachments/files/18246836/Gw2-64_d3d11.log)
[Gw2-64_dxgi.log](https://github.com/user-attachments/files/18246840/Gw2-64_dxgi.log)
[vulkaninfo.log](https://github.com/user-attachments/files/18246841/vulkaninfo.log)
[wine_buildin.log](https://github.com/user-attachments/files/18246842/wine_buildin.log)
[wine_native.log](https://github.com/user-attachments/files/18246843/wine_native.log)

**Comment by Blisto91** [comment]
Does it work if you switch the game to run in windowed mode without dxvk and then start it again with dxvk?

**Comment by jotkah** [comment]
Yay - that work's. Thank you so much. Seems to be a "full screen issue".

For me, this is a practicable work-around which i will keep in mind for future. Do i have a fps loss when i run in windowed mode instead of fullscreen mode?

**Comment by Blisto91** [comment]
I don't believe so as Linux doesn't have a concept of a exclusive fullscreen like there exists on Windows.

In regards to the issue itself i am thinking they may be a Nvidia driver issue. There at least have been problems with fullscreen games in regular system Wine with the newer driver versions. Though i have only tested d3d9 games in regular Wine when it was a issue. I am also unsure if it is still an issue with the latest 565 driver.

**Comment by Blisto91** [comment]
@jotkah Could you give this a retest with Nvidia driver 570 or above? I gather this issue *might* be gone.

**Comment by Blisto91** [comment]
I believe this issue should be gone with latest drivers. Feel free to comment here again if it still happens
